Strato/Plist2HTML.swift

```diff
@@ -0,0 +1,84 @@
+import Foundation
+
+enum Plist2HTML {
```

Comment:
I think we should rename it to something more precise, like `AcknowledgmentsPlistToHTML`, or `PodLicensesPlistToHTML`.